set.mm

```diff
@@ -382517,9 +382532,29 @@ proved using the inner form (see theorem 9.6 of [Schwabhauser] p. 69
 
   ${
     $d a b g i j $.
-    $( Define the relation congruence bewteen shapes.  Definition 4.4 of
-       [Schwabhauser] p. 35.  Ideally, we would define this for functions of
-       any set, but we will used words (functions over ` NN ` ) in most cases.
+    $( Define the relation of congruence between shapes.  Definition 4.4 of
+       [Schwabhauser] p. 35.  A "shape" is a finite sequence of points, and a
+       triangle can be represented as a shape with three points.  Two shapes
+       are congruent if all corresponding segments between all corresponding
+       points are congruent.
+
+       Many systems of geometry define triangle congruence as requiring both
+       segment congruence and angle congruence.  Such systems, such as
+       Hilbert's axiomatic system, typically have a primitive notion of angle
+       congruence in addition to segment congruence.  Here, angle congruence is
+       instead a derived notion, defined later in ~ df-cgra and expanded in
+       ~ iscgra .  This does not mean our system is weaker; ~ dfcgrg2 proves
+       that these two definitions are equivalent, and using the Tarski
+       definition instead (given in [Schwabhauser] p. 35) is simpler.  Once two
+       triangles are proven congruent as defined here, you can use various
+       theorems to prove that corresponding parts of congruent triangles are
+       congruent (CPCTC).  For example, see ~ cgr3simp1 , ~ cgr3simp2 ,
+       ~ cgr3simp3 , ~ cgrcgra , and permutation laws such as ~ cgr3swap12 and
+       ~ dfcgrg2 .
+
+       Ideally, we would define this for functions of any set, but we will use
+       words (functions over ` NN ` ) in most cases.
```

Comment:
Actually, words are defined as functions over `NN0`, not over `NN`.

Reply:
well... functions over `( 0 ..^ n )` for some n \in NN0 !

Reply:
Indeed! Maybe just refer to ~ df-word ?
Something like 
> but we will use words (see ~ df-word) in most cases.

Reply:
I like that, it's both clear and accurate (which is often hard to do). I'll post an update with that.